commands/upload/upload.go

```diff
@@ -132,7 +132,7 @@ func runProgramAction(pm *packagemanager.PackageManager,
 	// Find target board and board properties
 	_, boardPlatform, board, boardProperties, buildPlatform, err := pm.ResolveFQBN(fqbn)
 	if err != nil {
-		return fmt.Errorf("incorrect FQBN: %s", err)
+		return fmt.Errorf("Error resolving FQBN: %s", err)
```

Comment:
```suggestion
		return fmt.Errorf("incorrect FQBN: %s", err)
```

I would leave it as it is, that part is clear.

Reply:
Commit dropped